Core/Utilities/ElasticSearchDB.py

```diff
@@ -101,7 +109,9 @@ def getIndexes( self ):
     """
     It returns the available indexes...
     """
-    return [ index for index in self.__client.indices.get_aliases() ]
+    #we only return indexes which belong to a specific setup for example 'lhcb-production' or 'dirac-production etc.
+    #this is a policy used by IT to distinguish between users.
```

Comment:
This is DIRAC, please don't refer specifically to "IT" anywhere because it makes no sense here.